mllib/src/main/scala/org/apache/spark/mllib/tree/RandomForest.scala

```diff
@@ -23,6 +23,7 @@ import scala.util.Try
 import org.apache.spark.annotation.Since
 import org.apache.spark.api.java.JavaRDD
 import org.apache.spark.internal.Logging
+import org.apache.spark.ml.feature.{Instance => NewInstance}
```

Comment:
Why rename this import? I'm probably missing it

Reply:
I think it is just following the pattern of the rest of the code, all of the ML-related (as opposed to MLLIB) is renamed, for example see:

import org.apache.spark.ml.tree.{DecisionTreeModel => NewDTModel, TreeEnsembleParams => NewRFParams}
import org.apache.spark.ml.tree.impl.{RandomForest => NewRandomForest}

This isn't a deal-breaker for me though so I've renamed it back